_data/simple-icons.json

```diff
@@ -1665,6 +1665,11 @@
             "hex": "2B579A",
             "source": "https://www.office.com"
         },
+        {
+            "title": "Mint",
```

Comment:
Looks like they're using the full "Linux Mint" whenever they refer to themselves

Reply:
It was my mistake, and now it has been corrected. Thank you @birjolaxew for your code review 😃